x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality_file_uploader/components/risk_engine_callout.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiHorizontalRule,
+  EuiText,
+  EuiFlexItem,
+} from '@elastic/eui';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { i18n } from '@kbn/i18n';
+import { useAppToasts } from '../../../../common/hooks/use_app_toasts';
+import { formatScheduledRunTime } from '../helpers';
+import { useScheduleNowRiskEngineMutation } from '../../../api/hooks/use_schedule_now_risk_engine_mutation';
+import {
+  useInvalidateRiskEngineStatusQuery,
+  useRiskEngineStatus,
+} from '../../../api/hooks/use_risk_engine_status';
+
+const TEN_SECONDS = 10000;
+
+export const RiskEngineCallout: React.FC = () => {
+  const { data: riskEngineStatus, isLoading: isRiskEngineStatusLoading } = useRiskEngineStatus();
+  const { addSuccess, addError } = useAppToasts();
+  const { isLoading: isLoadingRiskEngineSchedule, mutate: scheduleRiskEngineMutation } =
+    useScheduleNowRiskEngineMutation({
+      onSuccess: () =>
+        addSuccess(
+          i18n.translate(
+            'xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.successMessage',
+            {
+              defaultMessage: 'Risk engine run scheduled',
+            }
+          )
+        ),
+      onError: (error) =>
+        addError(error, {
+          title: i18n.translate(
+            'xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.errorMessage',
+            {
+              defaultMessage: 'Risk engine schedule failed',
+            }
+          ),
+        }),
+    });
+  const [nextScheduleRun, setNextScheduleRun] = useState<string | undefined>();
+  const invalidateRiskEngineStatusQuery = useInvalidateRiskEngineStatusQuery();
+  const { status, runAt } = riskEngineStatus?.risk_engine_task_status || {};
+
+  const isRunning = useMemo(
+    () => status === 'running' || (!!runAt && new Date(runAt) < new Date()),
+    [runAt, status]
+  );
+
+  const updateCountDownText = useCallback(() => {
+    if (isRunning) {
+      setNextScheduleRun('Now running');
+    } else {
+      setNextScheduleRun(formatScheduledRunTime(riskEngineStatus?.risk_engine_task_status?.runAt));
+    }
+  }, [isRunning, riskEngineStatus?.risk_engine_task_status?.runAt]);
+
+  useEffect(() => {
+    updateCountDownText();
+    const intervalId = setInterval(() => {
```

Comment:
Could the polling logic be simplified by using the react query `refetchInterval` parameter and just poll regardless of whether the risk engine is running or not?